EnergyAndMaterialBalanceModule/Data/Repositories/BGroupsRepository.cs

```diff
@@ -22,5 +20,10 @@ public IEnumerable<Bgroups> GetRootBGroups(int resourceId)
         {
             return Context.Bgroups.Where(t => t.ResourceId == resourceId && t.BgroupIdparent == null).AsEnumerable();
         }
+
+        public IEnumerable<Bgroups> GetChildren(int groupid)
+        {
+            return Context.Bgroups.Where(t => t.BgroupIdparent.HasValue && t.BgroupIdparent.Value == groupid).AsEnumerable();
```

Comment:
We need something like 
```suggestion
            return Context.Bgroups.Where(t => t.BgroupId == id).Include(f => f.InverseBgroupIdparentNavigation).First();
```

Reply:
don`t get this. By name in my case you get all groups having selected as a parent
in your case you just do the same thing as get by id but include parent into it. what goes against the name of method
I thought we use lazy loading, cause thought it is default config for ef.

Reply:
also in my case, there is no need for include